sentinel-adapter/sentinel-jdbc-adapter/src/main/java/com/alibaba/csp/sentinel/adapter/jdbc/JDBCContext.java

```diff
@@ -0,0 +1,32 @@
+package com.alibaba.csp.sentinel.adapter.jdbc;
+
+
+/**
+ * @author icodening
+ * @date 2022.02.09
+ */
+public interface JDBCContext {
```

Comment:
Could you please add some javadocs for the interface (and methods)?

Reply:
OK, i will slove it